plugin/validator_test.go

```diff
@@ -956,6 +956,889 @@ func TestValidator_SchemaAsJSON(t *testing.T) {
+		{
+			name: "valid hosts with wildcards",
+			config: `{
+				"hosts": [
+					"192.0.2.1",
+					"*.konghq.com",
+					"konghq.*"
+				],
+				"tag": "test"
+			}`,
+		},
+		{
+			name: "valid hosts with wildcards",
+			config: `{
+				"hosts": [
+					"192.0.2.1",
+					"*konghq.com",
+					"*",
+					"konghq*",
+					"*.konghq.*",
+					"konghq.*.com"
+				],
+				"tag": "test"
+			}`,
+			wantErr: true,
+			expectedErr: `{"config":{"hosts":[
+				"[2] = invalid wildcard: must be placed at leftmost or rightmost label",
+				"[3] = invalid wildcard: must be placed at leftmost or rightmost label",
+				"[4] = invalid wildcard: must be placed at leftmost or rightmost label",
+				"[5] = invalid wildcard: must have at most one wildcard",
+				"[6] = invalid wildcard: must be placed at leftmost or rightmost label"
+				]}}`,
+		},
+		{
+			name: "valid semantic_version",
+			config: `{
+				"semantic_version": "1",
+				"tag": "test"
+			}`,
+		},
+		{
+			name: "valid semantic_version",
+			config: `{
+				"semantic_version": "1.2",
+				"tag": "test"
+			}`,
+		},
+		{
+			name: "valid semantic_version",
+			config: `{
+				"semantic_version": "1.2.3",
+				"tag": "test"
+			}`,
+		},
+		{
+			name: "valid semantic_version",
+			config: `{
+				"semantic_version": "100.200.300",
+				"tag": "test"
+			}`,
+		},
+		{
+			name: "valid semantic_version",
+			config: `{
+				"semantic_version": "1.2.3-rc.1",
+				"tag": "test"
+			}`,
+		},
+		{
+			name: "valid semantic_version",
+			config: `{
+				"semantic_version": "1.2.3-alpha.1",
+				"tag": "test"
+			}`,
+		},
+		{
+			name: "valid semantic_version",
+			config: `{
+				"semantic_version": "1.2.3-beta.1",
+				"tag": "test"
+			}`,
+		},
+		{
+			name: "valid semantic_version",
+			config: `{
+				"semantic_version": "1.2.3.4-enterprise-edition",
+				"tag": "test"
+			}`,
+		},
+		{
+			name: "invalid semantic_version",
+			config: `{
+				"semantic_version": "hello",
+				"tag": "test"
+			}`,
+			wantErr:     true,
+			expectedErr: `{"config":{"semantic_version":"invalid version number: must be in format of X.Y.Z"}}`,
+		},
+		{
+			name: "invalid semantic_version",
+			config: `{
+				"semantic_version": ".1",
+				"tag": "test"
+			}`,
+			wantErr:     true,
+			expectedErr: `{"config":{"semantic_version":"invalid version number: must be in format of X.Y.Z"}}`,
+		},
+		{
+			name: "invalid semantic_version",
+			config: `{
+				"semantic_version": "1..1",
+				"tag": "test"
+			}`,
+			wantErr:     true,
+			expectedErr: `{"config":{"semantic_version":"must not have empty version segments"}}`,
+		},
+		{
+			name: "lua code not accepted",
+			config: `{
+				"lua_code": {"header": "return nil"},
+				"tag": "test"
+			}`,
+			wantErr: true,
+			expectedErr: `{"config":{
+				"lua_code":"Error parsing function: lua-tree/share/lua/5.1/kong/tools/sandbox.lua:119: ` +
+				`loading of untrusted Lua code disabled because 'untrusted_lua' config option is set to 'off'"
+				}}`,
+		},
```

Comment:
Since the sandbox is enabled by default (if I am not mistaken) we should also create a test that is valid for the sandbox; see https://docs.konghq.com/hub/kong-inc/serverless-functions/#sandboxing.

In the future we may need a way to configure this option in goks.

Reply:
By inspecting `configuration.enabled` and `configuration.sandbox_enabled` [here](https://github.com/Kong/goks/blob/main/lua-tree/share/lua/5.1/kong/tools/sandbox.lua#L118), both are `nil`, which makes the code fail as in this test.

Am I missing something?